go.mod

```diff
@@ -3,21 +3,21 @@ module github.com/giantswarm/kvm-operator
 go 1.15
 
 require (
-	github.com/giantswarm/apiextensions/v3 v3.33.0
+	github.com/giantswarm/apiextensions/v3 v3.39.0
 	github.com/giantswarm/badnodedetector v1.0.1
 	github.com/giantswarm/certs/v3 v3.1.1
 	github.com/giantswarm/errors v0.3.0
 	github.com/giantswarm/k8sclient/v5 v5.11.0
-	github.com/giantswarm/k8scloudconfig/v10 v10.14.0
+	github.com/giantswarm/k8scloudconfig/v11 v11.1.1
```

Comment:
I'm a little worried about this change. `k8scloudconfig` v11 introduces a breaking change: `Add feature to force cgroup v1 on Flatcar 3033.2.0 and above. This will not work with older Flatcar version.`. Are we ready for Flatcar 3033.2?